### ISSUE TITLE
Content check - description of force on mass spheres question

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#190]
@arouinfar, I was reviewing the descriptions of Gravity Force Lab Basics together with the visuals the other day, and I am wondering if I got things mixed up at some point, or perhaps only chose a phrase that results in an unnecessary change in word order that is opposite to the visual word order. 

Notice the word order in the second bullet describing the size of the force on each of the mass spheres.

Perhaps it is because in the visual the actual words are labeling the force, and in the description we are describing the spheres. Maybe it is all correct? Please verify.

<img width="1058" alt="Screen Shot 2019-12-10 at 8 33 30 PM" src="https://user-images.githubusercontent.com/1902116/70579887-7b866c80-1b8c-11ea-9e85-7a20f1b6c8ed.png">

**Comment by terracoda** [comment]
I update the image so both descriptions are showing in the screen shot.

**Comment by terracoda** [comment]
Apologies if this just a momentary laps in reason ;-)

**Comment by arouinfar** [comment]
@terracoda good catch. It looks the 2nd bullet point for mass 1 and mass 2 are indeed flipped. We should be describing the force on that particular object. So for mass 1, the 2nd bullet should read: 

* Size of force on mass 1 by mass 2 is 33.4 newtons.

The string should exactly match the one in the visual sim, but with a "size of" appended to the front (since this bullet point is describing the magnitude of the force).

**Comment by terracoda** [comment]
@arouinfar, so just to be clear the second bullets under the spheres should be like this:
‪
**mass 1‬, Blue Sphere**
_- Size of force **on ‪mass 1‬ by ‪mass 2**‬ is 33.4 ‪newtons‬._
‪
‪**mass 2‬, Red Sphere**
_- Size of force on ‪**mass 2‬ by ‪mass 1**‬ is 33.4 ‪newtons._‬

And this sim is already published, so could you advise how urgent this is for a maintenance release? Can it wait for January when we publish GFL regular, for example?

**Comment by arouinfar** [comment]
@terracoda yes, that all looks good to me.

With winter break coming up, there probably won't be much sim use in the coming weeks, so waiting until next month sounds reasonable. However, it also seems like a minor enough change (swapping 2 strings) to be done in a quick maintenance release. I'll defer to @emily-phet and @ariel-phet to decide the priority on this.

**Comment by terracoda** [comment]
@zepumph, this is an issue in both Basics and Regular!

**Comment by ariel-phet** [comment]
This issue seems worth a patch, my guess is it would be a fairly straightforward maintenance release with minimal testing required. @zepumph can let us know if otherwise.

**Comment by zepumph** [comment]
This has been fixed on master in both sims. I can cherry-pick it into the GFLB branch next though right now my priority is GFL, trying to publish in the next two weeks. @emily-phet please let me know if you would rather me work on this that GFL. It will be likely ~one hour.

**Comment by emily-phet** [comment]
@zepumph - yes, please go ahead with this.

**Comment by zepumph** [comment]
This looks like it is only a bug with master, and not with published version of the sim. It has to do with a refactor done after GFLB shas were taken. Here is a picture of the PDOM on the published version:

![image](https://user-images.githubusercontent.com/6856943/71631665-5480f080-2bb6-11ea-893a-80d14a06343e.png)


And here is the published version's a11y view: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0/phet/gravity-force-lab-basics_a11y_view.html

**Comment by zepumph** [comment]
I'm ready to close this issue, but I also want to let everyone see and have a chance to comment.

**Comment by terracoda** [comment]
Thanks for checking @zepumph.
Now, it makes sense why neither me nor QA caught this mistake before publishing. It wasn't there!
That's a relief?

**Comment by zepumph** [comment]
Alright, I'm going to close this.
